src/compiler.jl

```diff
@@ -3414,10 +3414,6 @@ nfields(Type::LLVM.VectorType) = size(Type)
 nfields(Type::LLVM.ArrayType) = length(Type)
 nfields(Type::LLVM.PointerType) = 1
 
-mutable struct EnzymeTape{N, T}
-    data::NTuple{N, T}
-end
-
 mutable struct EnzymeTapeToLoad{T}
```

Comment:
Also delete this then?

Reply:
No EnzymeTapeToLoad is distinct and used in split mode contexts by Enzyme internally to say if it is recursive and we need to load the tape for the reverse pass before the call